wiki/Beatmapping/en.md

```diff
@@ -1,337 +1,56 @@
+As _osu!_ is a rhythm based game, the song/music you choose to map should have an obvious and constant beat.
+Try to avoid songs which have tempo (or speed) changes or slow songs until you are more experienced and used to beatmapping.
 
-The most common difficulties created in Beatmapping are:
+## Getting Started
 
--   [Easy](Easy_(Difficulty) "wikilink")
--   [Normal](Normal "wikilink")
--   [Hard](Hard "wikilink")
--   [Insane](Insane "wikilink")
--   [Expert](Expert "wikilink")
+To make a new beatmap, go to the edit mode then drag-and-drop an audio file into _osu!_.
+(Alternatively, you can move an audio file onto the `Songs` folder.)
+This should then open and direct you to the edit screen with your audio file.
+Your new song will be at the bottom of the song list, highlighted in purple.
+If it is not listed here for any reason, press `F5` to reset the beatmap cache and it should appear.
+Click on the song, this will open the [beatmap editor](/wiki/beatmap_editor) and the `Song Setup` dialog.
 
-Clicking each link will display a common set of guidelines [mappers](Mappers "wikilink") follow to be able to create an entertaining beatmap appropriate for its [difficulty](Difficulty "wikilink") level.
+### Song Setup
 
-It is also not uncommon for [mappers](Mappers "wikilink") to use custom [difficulty](Difficulty "wikilink") names in place of these. (Like "Mario" for example)
+_See also: [Song Setup](/wiki/Song_Setup)._
 
-Additionally you may find [Guest Difficulties](Guest_Difficulty "wikilink") which are commonly labeled as mapper's difficulty (For example: "Larto's Insane").
+When making a new [mapset](/wiki/mapset), you will always see this dialog before any other parts of the beatmap editor.
 
-With the addition of the Special modes, you can now add mod-specific difficulties to your beatmap other than the usual osu! difficulty. It is entirely possible to fill the beatmap with only Taiko/Catch the Beat/Osu!mania difficulties provided there is an Easy/Normal difficulty in it.
+You should fill out this information before you create another [difficulty](/wiki/difficulty).
+This will ensure that the metadata will be consistent across all of the difficulties in the set.
 
-Getting Started
-===============
+You can change the options in the other tabs now if you want, but they can be changed later on.
+Click `OK`, and you can start beatmapping!
+The next step is to add timing to your beatmap.
 
-How do I start a new beatmap?
------------------------------
+### Timing
 
-As osu! is a rhythm based game, the songs you choose to map should have an obvious and constant beat. Try to avoid songs which have tempo (speed) changes or slow songs until you are more experienced and used to the beatmapping mechanism.
+_See also: [Timing](/wiki/Timing)._
 
-To start a new beatmap, follow these simple steps:-
+**Timing is vital!**
+An incorrectly timed maps will not be considered for ranking.
```

Comment:
An incorrectly timed maps?
`Incorrectly timed maps will not be considered for ranking.`